README.md

```diff
@@ -4,17 +4,17 @@
 
 # flathub.org
 
-This is the source code for the website, both backend and frontend. That's running on https://flathub.org
+This is the source code for the website, both backend and frontend. That's running on <https://flathub.org>.
```

Comment:
Why do you markup links like this?

Reply:
In the [CommonMark Specification](https://commonmark.org/), generally, links in between texts are enclosed in [angular brackets](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses) to be highlighted.

The angular bracket is optional here, as GitHub automatically highlights URLs automatically based on the presence of `https` or `http` tags before URLs, I have added them before the localhost URL in order to make it clickable.

Reply:
Hrm, in vscode those urls are already clickable as is

Reply:
Done, reverted this change.